float32 bending_compliance
---
bool success
